testbed/testbed/mock_backend.go

```diff
@@ -58,13 +50,14 @@ const (
 	BackendOC
 )
```

Comment:
It seems BackendType is not used anymore. Can it be removed?

Reply:
Thanks for the catch. Removed.